src/components/MultiSelect/MultiSelect-story.js

```diff
@@ -81,8 +81,58 @@ storiesOf('MultiSelect', module)
         label="Label"
         items={items}
         itemToString={item => (item ? item.text : '')}
-        onChange={action('onChange - Inline MultiSelect')}
+        onChange={action('onChange')}
         disabled
       />
     )
+  )
+  .addWithInfo(
+    'with initial selected items',
+    `
+      Provide a set of items to initially select in the control
+    `,
+    () => (
+      <div style={{ width: 300 }}>
+        <MultiSelect
+          label="Label"
+          items={items}
+          itemToString={item => (item ? item.text : '')}
+          initialSelectedItems={[items[0], items[1]]}
+          onChange={action('onChange - Inline MultiSelect')}
+        />
+      </div>
+    )
+  )
+  .addWithInfo(
+    'filterable',
+    `
+      Filterable version of our MultiSelect component
+    `,
+    () => (
+      <div style={{ width: 300 }}>
+        <MultiSelect.Filterable
+          items={items}
+          itemToString={item => (item ? item.text : '')}
+          onChange={action('onChange')}
+          placeholder="Placeholder..."
```

Comment:
Very minor thing - but do you want to check with @bsonefeld what this placeholder should say? We try and keep the design kit and the coded components in sync as much as possible, and I _think_ it just says `Filter` in the design kit but I might be wrong!

Reply:
Cool! Makes sense to me. Should I just grab the placeholder from the dropdown stuff?

Reply:
Yeah, either just grab it from the designs or ask one of the designers directly! 🙂

Reply:
Yep, @marijohannessen 's right, the placeholder should say `Filter`. 

Ref: 
![screen shot 2018-01-19 at 2 19 46 pm](https://user-images.githubusercontent.com/11233508/35169893-d3ee0916-fd23-11e7-9e08-17e6eaee6139.png)

Reply:
Updated!